src/interface/src/app/treatments/map-actionbutton/map-actionbutton.component.html

```diff
@@ -0,0 +1,12 @@
+<mgl-control position="top-right">
```

Comment:
Can you rename these files to map-action-button ? Seems like we are missing a `-`